integration_tests/sdk/utils.py

```diff
@@ -193,7 +193,7 @@ def delete_flow(client: aqueduct.Client, workflow_id: uuid.UUID) -> None:
     try:
         client.delete_flow(str(workflow_id))
     except Exception as e:
-        print("Error deleting workflow %s with exception %s" % (workflow_id, e))
+        print("Error deleting workflow %s with exception: %s" % (workflow_id, e))
```

Comment:
try to use `%v` for errors, this might give you more information

Reply:
Was looking at Python formatting reference and I don't think `%v` exists in Python. https://python-reference.readthedocs.io/en/latest/docs/str/formatting.html

Ran a small example:
```
try:
    1 / 0
except Exception as e:
    print("%s" % e)
    print("%r" % e)
    print("%v" % e)
```
Output:
```
division by zero
ZeroDivisionError('division by zero')
Traceback (most recent call last):
  File "main.py", line 2, in <module>
    1 / 0
ZeroDivisionError: division by zero
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "main.py", line 6, in <module>
    print("%v" % e)
ValueError: unsupported format character 'v' (0x76) at index 1
```

I can do something like in https://github.com/aqueducthq/aqueduct/pull/336 to add the exception class' name.

Reply:
oh nvm I mixed go with python here